terraform/README.md

```diff
@@ -0,0 +1,65 @@
+# Grafana-agent-k8s Terraform Module
+
+This Grafana-agent-k8s Terraform module aims to deploy the [grafana-agent-k8s charm](https://charmhub.io/grafana-agent-k8s) via Terraform.
+
+## Getting Started
```

Comment:
In general, I'm not sure if this section is needed, since the module is intended to be imported into a higher level module. WDYT?

Reply:
This is removed.